command/meta.go

```diff
@@ -36,9 +36,9 @@ type Meta struct {
 
 	// Variables for the context (private)
 	autoKey       string
-	autoVariables map[string]string
+	autoVariables map[string]interface{}
```

Comment:
Yes - these are variables loaded from `terraform.tfvars` irrespective of whether a `-var-file` is passed.